gradio/networking.py

```diff
@@ -231,9 +231,11 @@ def serve_forever():
     return httpd
 
 
-def start_simple_server(interface, directory_to_serve=None, server_name=None):
+def start_simple_server(interface, directory_to_serve=None, server_name=None, server_port=None):
+    if server_port is None:
+        server_port = INITIAL_PORT_VALUE
```

Comment:
if the default value of None gets set to INITIAL_PORT_VALUE, you could set that as the default in the first place